helmchart/otel-add-on/values.schema.json

```diff
@@ -0,0 +1,268 @@
+        "annotations",
+        "name"
+      ]
+    },
+    "podAnnotations": {
+      "type": "object",
+      "properties": {},
+      "required": []
+    },
+    "podLabels": {
+      "type": "object",
+      "properties": {},
+      "required": []
+    },
+    "podSecurityContext": {
+      "type": "object",
+      "properties": {},
+      "required": []
+    },
+    "securityContext": {
+      "type": "object",
+      "properties": {
+        "capabilities": {
+          "type": "object",
+          "properties": {
+            "drop": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          },
+          "required": [
+            "drop"
+          ]
+        },
+        "readOnlyRootFilesystem": {
+          "type": "boolean"
+        },
+        "runAsNonRoot": {
+          "type": "boolean"
+        },
+        "runAsUser": {
+          "type": "number"
+        }
+      },
+      "required": [
+        "capabilities",
+        "readOnlyRootFilesystem",
+        "runAsNonRoot",
+        "runAsUser"
+      ]
+    },
+    "service": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string"
+        },
+        "otlpReceiverPort": {
+          "type": "number"
+        },
+        "kedaExternalScalerPort": {
+          "type": "number"
+        }
+      },
+      "required": [
+        "type",
+        "otlpReceiverPort",
+        "kedaExternalScalerPort"
+      ]
+    },
+    "resources": {
+      "type": "object",
+      "properties": {
+        "limits": {
+          "type": "object",
+          "properties": {
+            "cpu": {
+              "type": "string"
+            },
+            "memory": {
+              "type": "string"
+            }
+          },
+          "required": [
+            "cpu",
+            "memory"
+          ]
+        },
+        "requests": {
+          "type": "object",
+          "properties": {
+            "cpu": {
+              "type": "string"
+            },
+            "memory": {
+              "type": "string"
+            }
+          },
+          "required": [
+            "cpu",
+            "memory"
+          ]
+        }
+      },
+      "required": [
+        "limits",
+        "requests"
+      ]
+    },
+    "volumes": {
+      "type": "array",
+      "items": {}
+    },
+    "volumeMounts": {
+      "type": "array",
+      "items": {}
+    },
+    "nodeSelector": {
+      "type": "object",
+      "properties": {},
+      "required": []
+    },
+    "tolerations": {
+      "type": "array",
+      "items": {}
+    },
+    "affinity": {
+      "type": "object",
+      "properties": {},
+      "required": []
+    },
+    "opentelemetry-collector": {
+      "type": "object",
+      "$ref": "https://raw.githubusercontent.com/open-telemetry/opentelemetry-helm-charts/refs/tags/opentelemetry-operator-0.74.2/charts/opentelemetry-collector/values.schema.json#/properties"
```

Comment:
we can reference the upstram json schema for otel collector sub-chart

Reply:
and I guess this version is a different version from chart dependency so they don't need to match?

```
repository: https://open-telemetry.github.io/opentelemetry-helm-charts
version: 0.110.0
```
vs 
```
opentelemetry-operator-0.74.2/charts/opentelemetry-collector
```

Reply:
Thanks, I was trying to spot this tag in that repo, but didn't find it. But it is actually there.